tuned/plugins/plugin_cpu.py

```diff
@@ -203,11 +232,31 @@ def _instance_apply_static(self, instance):
 	def _instance_unapply_static(self, instance, full_rollback = False):
 		super(CPULatencyPlugin, self)._instance_unapply_static(instance, full_rollback)
 
+		if len(self._reenable_thread_siblings) > 0:
+			log.info("re-enabling thread siblings: %s" % str(self._reenable_thread_siblings))
+			for c in self._reenable_thread_siblings:
+				self._cmd.write_to_file("/sys/devices/system/cpu/cpu%d/online" % c, "1")
+			self._reenable_thread_siblings= []
+			self._storage.set(self._thread_siblings_storage_key, self._reenable_thread_siblings)
+
 		if instance._first_instance and self._has_intel_pstate:
 			self._set_intel_pstate_attr("min_perf_pct", self._min_perf_pct_save)
 			self._set_intel_pstate_attr("max_perf_pct", self._max_perf_pct_save)
 			self._set_intel_pstate_attr("no_turbo", self._no_turbo_save)
 
+	def _instance_verify_static(self, instance, ignore_missing, devices):
+		ret1 = super(CPULatencyPlugin, self)._instance_verify_static(instance, ignore_missing, devices)
+		ret2 = True
+		offline_cpus = []
+		for c in self._reenable_thread_siblings:
+			if self._cmd.read_file("/sys/devices/system/cpu/cpu%d/online" % c).strip() == "0":
+				offline_cpus.append(c)
+		still_online = set(self._reenable_thread_siblings) - set(offline_cpus)
+		if len(still_online) > 0:
+			log.error("the following CPUs should be offline, but are not: %s" % str(still_online))
```

Comment:
Could you prepend the "verify: " prefix (i.e. for the log analyzers to easily find the verification output). Or better add the STR_VERIFY_PROFILE_.... consts. It should also log.info(), if the verification passes.

Reply:
Using the consts now.